src/components/search/index.js

```diff
@@ -4,12 +4,12 @@ import Box from '../box';
 import EmptyState from '../emptyState';
 import styles from './search.css';
 
-const SearchResult = ({ t }) => (
+const Search = ({ t }) => (
```

Comment:
`SearchResult` is IMO more descriptive for what this component does. So rather change the folder name